CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -0,0 +1,92 @@
+
+### Changed
+
+- Antrea now uses OpenFlow 1.5 to program OVS. ([#3770](https://github.com/antrea-io/antrea/pull/3770), [@wenyingd] [@ashish-varma])
+- Rename Windows script Start.ps1 to Start-AntreaAgent.ps1, and rename Stop.ps1 to Stop-AntreaAgent.ps1. ([#3904](https://github.com/antrea-io/antrea/pull/3904), [@wenyingd])
+- Unify NodePortLocal behavior across Linux and Windows. Linux agents now support allocating different Node ports for different protocols even when the Pod port number is the same. ([#3936](https://github.com/antrea-io/antrea/pull/3936), [@XinShuYang])
+- Antrea IPAM now uses the name of the uplink interface to name the host internal port, and the uplink interface will be renamed with a `~` suffix, e.g. `eth0~`. ([#3938](https://github.com/antrea-io/antrea/pull/3938), [@gran-vmv])
+- Send Neighbor Advertisement messages after creating Pods in an IPv6 cluster. ([#3998](https://github.com/antrea-io/antrea/pull/3998), [@gran-vmv])
+- Add an output formatter "raw" to better display multi-line string responses for antctl. ([#3589](https://github.com/antrea-io/antrea/pull/3589), [@Atish-iaf])
+- Add new ports to network requirement doc. ([#4063](https://github.com/antrea-io/antrea/pull/4063), [@luolanzone])
+- Windows OVS installation script now installs required SSL library if missing. ([#4029](https://github.com/antrea-io/antrea/pull/4029), [@XinShuYang])
+- Upgrade whereabouts CNI to v0.5.4 and provide required pluginArgs when invoking the CNI binary. ([#3987](https://github.com/antrea-io/antrea/pull/3987), [@arunvelayutham])
+- Remove Grafana flow collector files in the Antrea repo (as they were moved to the Theia repo). ([#4048](https://github.com/antrea-io/antrea/pull/4048), [@dreamtalen])
+- Make the following changes to the Multi-cluster feature:
+  * Add columns of kubectl outputs for Multi-cluster types. ([#3923](https://github.com/antrea-io/antrea/pull/3923), [@jianjuns])
+  * Use hostNetwork for Multi-cluster controller. ([#3965](https://github.com/antrea-io/antrea/pull/3965), [@luolanzone])
+  * Update ClusterClaim CRD to v1alpha2. ([#3755](https://github.com/antrea-io/antrea/pull/3755), [@bangqipropel])
+  * Update GatewayIPPrecedence to support the "external/internal" options. ([#3930](https://github.com/antrea-io/antrea/pull/3930), [@luolanzone])
+  * Disable metrics API and change the health binding address port to 8080. ([#4101](https://github.com/antrea-io/antrea/pull/4101), [@luolanzone])
+  * Improve CRD validation. ([#4062](https://github.com/antrea-io/antrea/pull/4062) [#4090](https://github.com/antrea-io/antrea/pull/4090) [#4043](https://github.com/antrea-io/antrea/pull/4043), [@luolanzone])
+  * Cleanup ResourceExport if exported Service has no available Endpoints. ([#4056](https://github.com/antrea-io/antrea/pull/4056), [@luolanzone])
```

Comment:
Cleanup -> Clean up

Reply:
the exported Service

Reply:
I feel this one can be viewed as a bug fix. @luolanzone ?

Reply:
Yeah, it can be moved to 'Fixed'

Reply:
done